### ISSUE TITLE
autoFocus and label's active class

### ISSUE THREAD

**Issue by DWboutin** [Dogfalo/materialize#2488]
Hi, I'm making an app with React JS and i have to do an autoFocus on the first field. When the autoFocus is made, the label don't go up with the active class.

![capture d ecran 2015-12-15 a 11 57 47](https://cloud.githubusercontent.com/assets/1322006/11817510/25aa8e44-a323-11e5-9cbc-9b4a256e4269.png)

When i put the active class on the label, it remove it because of this:

```
Materialize.updateTextFields = function() {
      var input_selector = 'input[type=text], input[type=password], input[type=email], input[type=url], input[type=tel], input[type=number], input[type=search], textarea';
      $(input_selector).each(function(index, element) {
        if ($(element).val().length > 0 || $(this).attr('placeholder') !== undefined || $(element)[0].validity.badInput === true) {
          $(this).siblings('label').addClass('active');
        }
        else {
          $(this).siblings('label, i').removeClass('active');
        }
      });
    };
```

Is there a way i can do this right? Thank you

**Comment by trostli** [comment]
i think there is a fix in PR #2374 . hopefully that gets merged in

**Comment by acburst** [comment]
Merged
